jsscripts/ContextMenuHandler.js

```diff
@@ -4,22 +4,23 @@
 
 const kXLinkNamespace = "http://www.w3.org/1999/xlink";
 
-let Node = Ci.nsIDOMNode;
-
 Logger.debug("JSScript: ContextMenuHandler.js loaded");
 
 var ContextMenuHandler = {
+  content: null,
   _types: [],
   _previousState: null,
 
-  init: function ch_init() {
+  init: function ch_init(content) {
+    this.content = content
     // Events we catch from content during the bubbling phase
-    addEventListener("contextmenu", this, false);
-    addEventListener("pagehide", this, false);
+    let rootWindow = content.windowRoot;
+    rootWindow.addEventListener("contextmenu", this, false);
+    rootWindow.addEventListener("pagehide", this, false);
 
     // Messages we receive from browser
     // Command sent over from browser that only we can handle.
-    addMessageListener("Browser:ContextCommand", this, false);
+    content.docShell.messageManager.addMessageListener("Browser:ContextCommand", this, false);
```

Comment:
After checking I think this is not needed / used at all. This piece  code was introduced in sha1 11ad36ab and I think it was really only used at  Metro back in the day. Hence, one cannot find "Browser:ContextCommand" event dispatching anywhere. Further, if you check what does that _onContextCommand do: cut, copy, paste, select-all, copy image =>  cut from editor is missing but I don't recall it ever being working. We have select-all implemented but not used by browser. Looks indeed void. Any thoughts or did I miss something?

Reply:
Added a commit on top that cleans this.